packages/shopify-codemods/src/mocha-context-to-closure.js

```diff
@@ -0,0 +1,185 @@
+import takeWhile from 'lodash/takeWhile';
```

Comment:
You don't need to compile codemods, as jscodeshift runs them with babel registered